### PR TITLE
update to be consistent with Patternfly guidelines

### PR DESCRIPTION
**Pull request by jamestalton**
Signed-off-by: James Talton <jtalton@redhat.com>

https://github.com/open-cluster-management/backlog/issues/12656

### CODE REVIEW

File: frontend/public/locales/en/cluster.json [hunk 1]
```diff
@@ -388,38 +388,38 @@
...
+    "bulk.title.detach": "Detach clusters?",
     "bulk.message.detach": "Detaching a cluster removes it from management, but does not destroy it. You can import the clusters again to manage them.",
-    "bulk.title.destroy": "Destroy cluster(s)",
-    "bulk.message.destroy": "Destroying a cluster is irreversible. This action detaches the cluster and if possible the cluster will be destroyed.",
-    "bulk.title.hibernate": "Hibernate cluster(s)",
-    "bulk.message.hibernate": "Moving to the Hibernating state will block any operations for the cluster(s). While hibernating, the cluster will not consume any virtual machine or network resources. This can be undone at any time.",
-    "bulk.title.resume": "Resume cluster(s)",
-    "bulk.message.resume": "Moving out of Hibernating state will resume all operations for the cluster(s). After powering back on, the cluster will resume consumption of any virtual machine and network resources.",
-    "bulk.title.removeSet": "Remove cluster from set",
+    "bulk.title.destroy": "Permanently sestroy clusters?",
```
Comment (showeimer) [inline frontend/public/locales/en/cluster.json:395]: destroy

File: frontend/src/routes/BareMetalAssets/BareMetalAsset.test.tsx [hunk 2]
```diff
@@ -141,7 +141,7 @@ describe('bare metal asset page', () => {
         await waitForNock(deleteNock)
     })
 
-    test('can delete asset(s) from batch action menu', async () => {
+    test('can delete assets from batch action menu', async () => {
```
Comment (showeimer) [inline frontend/src/routes/BareMetalAssets/BareMetalAsset.test.tsx:144]: 😂

File: frontend/public/locales/en/bma.json [hunk 3]
```diff
@@ -1,68 +1,67 @@
...
-  "modal.delete.content.batch": "Deleting a bare metal asset is irreversible.",
-  "modal.import.content.batch": "Importing bare metal assets will also create required namespaces.",
-  
+    "bareMetalAsset.importAction.title": "Specify CSV file",
+    "bareMetalAsset.importAction.button": "Open file",
+    "bareMetalAsset.importAction.message": "The first line of the CSV file must be a comma deliniated header row that defines the following columns: hostName, hostNamespace, bmcAddress, macAddress, role (optional), username, password.",
 
-  "bareMetalAsset.rowAction.deleteAsset.title":"Delete asset",
-  "bareMetalAsset.rowAction.editAsset.title":"Edit asset",
+    "modal.delete.title.batch": "Delete {{number}} bare metal assets",
```
Comment (showeimer) [inline frontend/public/locales/en/bma.json:51]: Why is this one a number?

**Comment by swopebe** [comment]
thanks for pulling me in, added some review comments, sorry took a couple days to get to this.

**Review by swopebe — CHANGES_REQUESTED** [review]

**Review by showeimer — APPROVED** [review]

**Pull request merged**